examples/metrics-config-datadog/main.tf

```diff
@@ -0,0 +1,21 @@
+# retrieve HVN ID of a previously created HVN
+# see https://registry.terraform.io/providers/hashicorp/hcp/latest/docs/data-sources/hvn
+data "hcp_hvn" "main" {
```

Comment:
Data lookup seems redundant given the output of this is the same as the id entered.